front/src/components/UserFormDesign.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from 'react';
+import { TextField, Button, CircularProgress, Alert, MenuItem, Select, FormControl, InputLabel, OutlinedInput, Chip, Box, Typography, Container, Paper } from '@mui/material';
+import { SelectChangeEvent } from '@mui/material/Select';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import { pink, teal } from '@mui/material/colors';
+
+// Custom theme
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: teal[500],
+    },
+    secondary: {
+      main: pink[500],
+    },
+  },
+});
+
+const levels = ['Easy', 'Normal', 'Hard'];
+
+const UserFormDesign: React.FC = () => {
+  const [name, setName] = useState('');
+  const [dob, setDob] = useState<string>('');
+  const [level, setLevel] = useState<string[]>([]);
```

Comment:
level is a string value not an array, it must only chose one value not multiple values!